seolondon/settings/__init__.py

```diff
@@ -197,6 +200,10 @@
             'PORT': os.environ['RDS_PORT'],
         }
     }
+elif 'DATABASE_URL' in os.environ:
```

Comment:
Im not sure what you mean sorry, when deploying to heroku usually you can just point the settings to PROD and thats that are you testing dev settings on heroku live?